_sass/page.scss

```diff
@@ -115,7 +115,6 @@ body {
 		a {
 			display: block;
 			padding: 10px 0;
-			decoration: none;
```

Comment:
what does this change do?

Reply:
The property (decoration) that is not in .scss lang has been removed. I don't know why it was added either.